sqrl-server/sqrl-server-core/src/main/java/com/datasqrl/graphql/server/RootGraphqlModel.java

```diff
@@ -148,20 +147,68 @@ public PostgresLogMutationCoords(String fieldName, String tableName, String inse
     }
 
     @Override
-    public DataFetcher<?> accept(MutationCoordsVisitor visitor) {
-      return visitor.visit(this);
+    public <R, C> R accept(MutationCoordsVisitor<R, C> visitor, C context) {
+      return visitor.visit(this, context);
     }
   }
 
+  @JsonTypeInfo(
+      use = JsonTypeInfo.Id.NAME,
+      property = "type"
+  )
+  @JsonSubTypes({
+      @Type(value = KafkaSubscriptionCoords.class, name = "kafka"),
+      @Type(value = PostgresSubscriptionCoords.class, name = "postgres_log")
+  })
+  public static abstract class SubscriptionCoords {
+    protected String type;
+    public abstract <R, C> R accept(SubscriptionCoordsVisitor<R, C> visitor, C context);
+    public abstract String getFieldName();
+  }
```

Comment:
This needs to be backwards compatible, if there is no type then we should default to kafka. We've had to deal with several issues with older configs not being backwards compatible.